GCRCatalogs/alphaq.py

```diff
@@ -21,33 +21,62 @@ def _subclass_init(self, filename, lightcone=True, **kwargs):
 
         assert os.path.isfile(filename), 'Catalog file {} does not exist'.format(filename)
         self._file = filename
-        self.lightcone = lightcone
+        majorVersion = 0
+        minorVersion = 0
+        if (majorVersion or minorVersion) and not kwargs.get('version', '').startswith('.'.join((majorVersion, minorVersion))):
```

Comment:
@dkorytov Line 26-27 need to be moved to after Line 38.

Reply:
That's right...:P